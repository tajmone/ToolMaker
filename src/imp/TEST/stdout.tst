%%PROCESS(,"stdout.tst.foo")
Nu skriver jag till stdout.tst.foo
ett par rader.
%%PROCESS()
Nu ställer jag tillbaks till stdout och läser stdout.tst.foo
%%PROCESS("stdout.tst.foo",)
