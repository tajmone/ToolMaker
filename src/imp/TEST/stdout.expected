########## stdout ##########
%%PROCESS(,"stdout.tst.foo")
Nu skriver jag till stdout.tst.foo
ett par rader.
%%PROCESS()
Nu ställer jag tillbaks till stdout och läser stdout.tst.foo
%%PROCESS("stdout.tst.foo",)
########## ../imp stdout.tst >stdout.tst.tmp; cat stdout.tst.tmp; rm stdout.tst.tmp ##########
Nu ställer jag tillbaks till stdout och läser stdout.tst.foo
Nu skriver jag till stdout.tst.foo
ett par rader.
